packages/airplane/internal/runtime/dev.ts

```diff
@@ -55,7 +60,40 @@ export const runtime: RuntimeInterface = {
   },
 
   sleep: async (duration: number): Promise<void> => {
-    return standardRuntime.sleep(duration);
+    const client = new Client();
+    const now = new Date();
+    const sleepID = await client.createSleep({
+      durationMs: duration,
+      until: calculateEndTimeISOString(now.getTime(), duration),
+    });
+    let isSleepFinished = false;
+    const controller = new AbortController();
+
+    const localSleep = new Promise<void>((resolve) => {
+      return sleepWithCancel(duration, controller).then(() => {
+        isSleepFinished = true;
+        resolve();
+      });
+    });
+
+    const checkSkip = async (id: string) => {
+      const poller = new Poller({ delayMs: 500 });
+      return poller.run(async () => {
```

Comment:
The return value is meaningless, so don't return to make that clear.

```suggestion
      await poller.run(async () => {
```